main.go

```diff
@@ -9,13 +9,16 @@ import (
 	"github.com/gunk/gunk/convert"
 	"github.com/gunk/gunk/format"
 	"github.com/gunk/gunk/generate"
+	"github.com/gunk/gunk/log"
 )
 
 var (
 	app = kingpin.New("gunk", "Gunk Unified N-terface Kompiler command-line tool.")
 
-	gen         = app.Command("generate", "Generate code from Gunk packages.")
-	genPatterns = gen.Arg("patterns", "patterns of Gunk packages").Strings()
+	gen                      = app.Command("generate", "Generate code from Gunk packages.")
+	genPatterns              = gen.Arg("patterns", "patterns of Gunk packages").Strings()
+	genDebugCommands         = gen.Flag("print-commands", "print the commands").Short('x').Bool()
+	genDebugPackageGenerated = gen.Flag("print-package-generated", "print the names of packages as they are generated").Short('v').Bool()
```

Comment:
Is there a way to not give long flag forms for these? I don't think anyone would use the long forms, and they're quite hard to remember anyway.

Reply:
Another option would be to leave `print-commands` alone and give the second one the long name `verbose`, which is more accurate.

Reply:
Not that I could make work. Not having a long name gives errors. I'll change to verbose though.

Reply:
It does look like kingpin enforces a long flag name for all flags. Oh well.